crates/astria-composer/Cargo.toml

```diff
@@ -50,6 +50,9 @@ tokio = { workspace = true, features = [
 tokio-util = { workspace = true, features = ["rt"] }
 tracing = { workspace = true, features = ["attributes"] }
 tryhard = { workspace = true }
+tonic = { workspace = true }
+tokio-stream = { workspace = true, features = ["net"] }
+tonic-health = "0.10.2"
```

Comment:
Might be worth moving it to the workspace as we standardize health service across grpc servers.